packages/jaeger-ui/tsconfig.json

```diff
@@ -12,6 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 {
-  "extends": "../../tsconfig",
-  "include": ["src/**/*.tsx", "typings"]
+  "extends": "../../tsconfig.json",
+  "include": ["src/**/*.tsx", "typings"],
+  "compilerOptions": {
+    "baseUrl": "..",
+    // other options...
+    "paths": {
+      "@jaegertracing/plexus": ["plexus/src"]
```

Comment:
Is this needed given the vite.config change? Doing override in tsconfig means it only affects typescript source files, but we have a mix of ts/js, so I was looking for a more global override.

Reply:
Yes, it is needed apparently it is required for the TypeScript compiler to need to know about our aliases so that it can compile all of our imported files accordingly.

Reply:
tsc-lint is failing
`
Error: [tsc-lint     ] packages/plexus/src/DirectedGraph/builtins/PureEdges.tsx(17,22): error TS6059: File '/home/runner/work/jaeger-ui/jaeger-ui/packages/plexus/src/DirectedGraph/builtins/EdgePath.tsx' is not under 'rootDir' '/home/runner/work/jaeger-ui/jaeger-ui/packages/jaeger-ui'. 'rootDir' is expected to contain all source files.
`

Reply:
> Is this needed given the vite.config change? Doing override in tsconfig means it only affects typescript source files, but we have a mix of ts/js, so I was looking for a more global override.

Just checked reverting the tsconfig to default, and hot reload is working fine. Had only made the changes because this website mentioned it https://dev.to/tilly/aliasing-in-vite-w-typescript-1lfo
Pushing the changes now.